docs/rackhd/configuration.rst

```diff
@@ -166,6 +167,8 @@ The following table describes the configuration parameters in config.json:
       - Address for RackHD WebSocket Service to bind to (defaults to '0.0.0.0').
     * - wssBindPort
       - Listening port for RackHD WebSocket Service (defaults to 9100).
+    * - trustedProxy
```

Comment:
dumb question - what's this default to? (i.e. what value is used if it's not explicitly in the monorail.json configuration file)